tox.ini

```diff
@@ -1,40 +1,73 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
-    py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
-# Note: if you add new Python versions, please add them to
-# [testenv:coverage] depends as well!
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    docs
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
-    coverage run -p -m zope.testrunner --test-path=src []
+    zope-testrunner --test-path=src []
 extras =
     test
-deps =
-    coverage
-setenv =
-     ZOPE_INTERFACE_STRICT_IRO=1
```

Comment:
I don't think we want to lose this.

Reply:
Agreed!

Reply:
Keeping it via https://github.com/zopefoundation/meta/pull/56